usaspending_api/etl/management/commands/submission_loader_wrapper.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import datetime
+import argparse
+
+from django.core.management.base import BaseCommand, CommandError
+from django.core.management import call_command
+
+
+from usaspending_api.common.validator.helpers import validate_boolean
+from usaspending_api.common.helpers.generic_helper import create_full_time_periods
+from usaspending_api.common.exceptions import InvalidParameterException
+
+logger = logging.getLogger('console')
+exception_logger = logging.getLogger("exceptions")
+
+
+class Command(BaseCommand):
+
+    def strtobool(arg):
+        try:
+            return validate_boolean(arg)
+        except InvalidParameterException:
+            raise argparse.ArgumentTypeError('Boolean value expected.')
+
+    def add_arguments(self, parser):
+        parser.add_argument('--all_time', action="store_true", help='all quarters and fiscal years.')
+        parser.add_argument('--ids', nargs=1, help='comma-separated list of ids to load', type=str)
```

Comment:
What was the justification for using a comma-separated string of IDs instead of passing a list of space separated IDs? The script wouldn't need to split the string on line 47 and argparse would ensure only integers are allowed.
Example:
`parser.add_argument('--ids', nargs='+', help='list of Submission IDs to re/load', type=int)`